docs/elasticsearch-docs.asciidoc

```diff
@@ -1,21 +1,41 @@
 [[elasticsearch-docs]]
 == Elasticsearch
 
-The documents produced by APM Server have a specific structure.
-There are examples of such documents as well as documentation 
-
+The APM Server produces documents with a specific structure.
+The defined structure is used to create an index in Elasticsearch.
+To see which fields are indexed in Elasticsearch, see {ref}/exported-fields.html[Exported Fields].
```

Comment:
@dedemorton it seems I am doing something wrong here with the linking, but I can't see what the issue is, could you have a look?

Reply:
sure...looking now...

Reply:
You still need to include fields.asciidoc. I just meant that you should remove the "Indexed field" section and intro. I didn't mean that you should remove the include directive. :-)

I think it would be better to add the include to the index.asciidoc file, though, rather than the bottom of elasticsearch-docs.asciidoc.

So add the following directive to line 30 in index.asciidoc:

include::./fields.asciidoc[]

Then you should be able to use: To see which fields are indexed in Elasticsearch, see <<exported-fields>>.